src/main/kotlin/org/rust/ide/annotator/RsCargoCheckAnnotator.kt

```diff
@@ -147,18 +147,23 @@ private fun filterMessage(file: PsiFile, document: Document, message: RustcMessa
     val spanFilePath = PathUtil.toSystemIndependentName(span.file_name)
     if (!file.virtualFile.path.endsWith(spanFilePath)) return null
 
+    @Suppress("NAME_SHADOWING")
     fun toOffset(line: Int, column: Int): Int? {
+        val line = line - 1
+        val column = column - 1
```

Comment:
Well, you could do it in Kotlin :) `` `line'` `` 

And actually for Haskell I would prefer if `ghc` didn't treat shadowing as a warning: this would greatly reduce the need for primes and the amount of bugs in my code :)